MAINTAINERS.yaml

```diff
@@ -123,33 +116,24 @@ cpu-kvm:
     - Andreas Sandberg <andreas.sandberg@arm.com>
 
 cpu-minor:
-  status: maintained
-  maintainers:
-    - Zhengrong Wang <seanyukigeek@gmail.com>
+  status: orphaned
 
 cpu-o3:
   status: orphaned
 
 cpu-simple:
-  status: maintained
-  maintainers:
+  status: orphaned
+  experts:
     - Jason Lowe-Power <jason@lowepower.com>
-    - Gabe Black <gabe.black@gmail.com>
 
 dev:
-  status: maintained
-  maintainers:
-    - Gabe Black <gabe.black@gmail.com>
+  status: orphaned
 
 dev-hsa:
-  status: maintained
-  maintainers:
-    - Matt Poremba <matthew.poremba@amd.com>
+  status: orphaned
```

Comment:
@abmerop, does it make sense to drop this tag? Do you want to continue to be the maintainer?